run/types/sessionIt.ts

```diff
@@ -56,3 +58,41 @@ export function bothPlatformsIt(
   // Define test for iOS
   mobileIt("ios", title, test);
 }
+
+// export function iosItWithSetup(
+//   title: string,
+//   test: (platform: SupportedPlatformsType) => Promise<void>
+// ) {
+//   const testName = `${title} ios`;
+
+//   return it(testName, async () => {
+//     console.info(`\n\n==========> Running "${testName}"\n\n`);
+//     await test("ios");
+//   });
+// }
+
+// Define the function to accept the title, a setup data object, and a test function that uses this object.
+export function iosItWithSetup(
```

Comment:
should be removed?

Reply:
I guess if we don't want to go down the path of setup data.. then yes?

Reply:
ok you can keep it but will need some cleaning up